includes/admin/class-wc-admin-assets.php

```diff
@@ -275,6 +275,17 @@ public function admin_scripts() {
 			}
 			if ( in_array( $screen_id, array( 'shop_coupon', 'edit-shop_coupon' ) ) ) {
 				wp_enqueue_script( 'wc-admin-coupon-meta-boxes', WC()->plugin_url() . '/assets/js/admin/meta-boxes-coupon' . $suffix . '.js', array( 'wc-admin-meta-boxes' ), WC_VERSION );
+				wp_localize_script(
+					'wc-admin-coupon-meta-boxes',
+					'woocommerce_admin_meta_boxes_coupon',
+					array(
+						'generate_button_text' => esc_html__( 'Generate coupon', 'woocommerce' ),
+						'characters'           => apply_filters( 'woocommerce_coupon_code_generator_characters', 'ABCDEFGHIJKLMNOPQRSTUVWXYZ012345678' ),
```

Comment:
Based on customer feedback, in AutomateWoo, we exclude ambiguous looking characters like `0 o i l 1` from generated coupon codes. The feedback we got was that these characters can be easily misinterpreted by users when manually typing codes.

Feel free to ignore, I randomly stumbled on this PR :)

Reply:
Perfect, thanks for the feedback and makes sense, will alter the PR to remove those characters.